### PR TITLE
Fix/issue low 18 reaudit

### PR DESCRIPTION
**Pull request by zakrad**

### CODE REVIEW

**Review by 0xSolDev — APPROVED** [review]

File: src/SelfPeggingAsset.sol [hunk 1]
```diff
@@ -1233,6 +1233,12 @@ contract SelfPeggingAsset is Initializable, ReentrancyGuardUpgradeable, OwnableU
         uint256 newRate = exchangeRateProviders[i].exchangeRate();
         TokenFeeStatus storage st = feeStatusByToken[i];
 
+        if (isInactive(st.raisedAt)) {
```
Comment (azat-hafizov) [inline src/SelfPeggingAsset.sol:1236]: multiplier will be stale if not reset here — we skip it on 1st pass after idle period but will hit on 2nd

File: test/SelfPeggingAsset.t.sol [hunk 2]
```diff
@@ -983,7 +983,7 @@ contract SelfPeggingAssetTest is Test {
         rETHExchangeRateProvider1.setExchangeRate(0.994e18);
         rETHExchangeRateProvider2.setExchangeRate(0.994e18);
 
-        vm.warp(block.timestamp + 2 days);
+        vm.warp(block.timestamp + 4 minutes);
```
Comment (azat-hafizov) [inline test/SelfPeggingAsset.t.sol:986]: please add tests with prolonged idle periods to verify it works in edge cases
- first swap after long inactivity — should be no spike
- second swap right after — no spike as well

**Review by 0xSolDev — APPROVED** [review]
